### DISCUSSION TITLE
IR Blaster Bricked ?

### DISCUSSION THREAD

**Discussion by alexantao** [arendst/Tasmota#21991] [Q&A]
Hi,
  I have a generic IR blaster with chip TYWE3S.
  I successfully flashed it with Tasmota using tuya-convert. After that, accessed and upgraded with tasmota-ir.bin OTA.
  Configured Wi-Fi and I was accessing it perfectly.

   My intention was to integrate to HA, and searching for information on how to configure it, I came to this page:
      [https://tasmota.github.io/docs/devices/YTF-IR-Bridge/#ir-codes](https://tasmota.github.io/docs/devices/YTF-IR-Bridge/#ir-codes)
  And based on this information, I put it on module  YTF IR Bridge, wich asks for 4 GPIO configurations (mine was 1, 2, 3 and 4).  I don't know what I exactly did, but I put those numbers in sequence, choosing the corresponding component from the table: 
    01 -> Led_i (number after that was 1)
    02 -> iRecev (no number)
    03 -> Button  (number 1)
     04 -> IRsend (number 8) 

After that, the device rebooted and I lost communication with it. Does not connects to wifi, does not open a WiFi hotspot. The blue light keeps on on a low level, not fully lit.

 I tried to reset it turning it on/off 10+ times, no juice. Tried to put it in pairing mode (10s button press), and also no juice.
 Also tried to plug again on computer to flash it again with tuya-convert, but it needs to be put on pairing mode, so I couldn't.

  Does anybody here knows if theres a way to restore it ? Do you think if I try to flash without tuya-convert I'll be able to ? 

Many Thanks !

**Comment by sfromis** [comment]
Not clear what you mean by that "asks for 4 GPIO configurations (mine was 1, 2, 3 and 4)", as this is not like how Tasmota handles setup. If what you mean is gpio 1-4, that's not likely to be a good config, as you need to use the gpios matching your hardware.

Tuya-convert (if working at all) can only be done once, not after the Tuya firmware was already replaced. Do check if your device is in AP mode.

While turning off/on 7 times (not 10) is a reasonable recovery technique, you should make sure to not be "too quick". I usually recommend disconnecting input power for two seconds, connecting for two seconds, and so on, 7 times.
